spring-integration-core/src/main/java/org/springframework/integration/leader/event/LeaderEventPublisher.java

```diff
@@ -45,4 +46,14 @@
 	 */
 	void publishOnRevoked(Object source, Context context, String role);
 
+	/**
+	 * Publish a failure to acquire lock event.
+	 *
+	 * @param source the component generated this event
+	 * @param context the context associated with event
+	 * @param role the role of the leader
+	 */
+	void publishOnFailedToAcquireLock(Object source, Context context, String role);
```

Comment:
Well, the `LeaderEventPublisher` is used not only from the `LockRegistryLeaderInitiator` and there is really might be some state in other implementation which may omit similar even.
I really would like to make it as generic as possible then, even if it is right now only for locks.
Therefore the name like `publishOnFailedToAcquire()` looks sufficient and enough to cover locks case as well.